libraries/AP_Motors/AP_Motors_Class.cpp

```diff
@@ -79,6 +79,10 @@ void AP_Motors::rc_write(uint8_t chan, uint16_t pwm)
         // we have a mapped motor number for this channel
         chan = _motor_map[chan];
     }
+    if (_pwm_mode == PWM_MODE_ONESHOT125) {
+        // OneShot125 uses a PWM range from 125 to 250 usec
+        pwm /= 8;
+    }
```

Comment:
Just to make sure, this won't bring any problem if someone enables oneshot125 in a Linux board? Or selecting it and not having motors in the FMU, but in IO?